service/src/test/resources/ehr/mapper/observation/example-observation-resource-12.json

```
@@ -0,0 +1,27 @@
+{
+  "resourceType":"Observation",
+  "id":"Consultation3-Topic2-Category-Examination-Observation-1",
+  "effectiveDateTime":"2019-06-01T10:30:00+00:00",
+  "issued":"2019-06-01T10:30:00+00:00",
+  "comment": "Test comment",
+  "referenceRange":[
+    {
+      "text":"Test reference range text",
+      "high":{
+        "value":20
+      }
+    }
+  ],
+  "code": {
+    "text": "test"
+  },
+  "interpretation": {
+    "coding": [
+      {
+        "system": "http://hl7.org/fhir/v2/0078",
+        "code": "H",
+        "display": "Very High Value"
+      }
+    ]
+  }
+}
```

The formatting is slightly off on parts of these json files
```suggestion
{
  "resourceType": "Observation",
  "id": "Consultation3-Topic2-Category-Examination-Observation-1",
  "effectiveDateTime": "2019-06-01T10:30:00+00:00",
  "issued": "2019-06-01T10:30:00+00:00",
  "comment": "Test comment",
  "referenceRange": [
    {
      "text": "Test reference range text",
      "high": {
        "value": 20
      }
    }
  ],
  "code": {
    "text": "test"
  },
  "interpretation": {
    "coding": [
      {
        "system": "http://hl7.org/fhir/v2/0078",
        "code": "H",
        "display": "Very High Value"
      }
    ]
  }
}
```

---

Fixed here c89a459